src/ircode.c

```diff
@@ -788,7 +788,7 @@ JL_DLLEXPORT jl_code_info_t *jl_uncompress_ir(jl_method_t *m, jl_code_instance_t
 
     jl_code_info_t *code = jl_new_code_info_uninit();
     uint8_t flags = read_uint8(s.s);
-    code->aggressive_constprop = !!(flags & (1 << 4));
+    code->constprop = !!(flags & (3 << 4));
```

Comment:
I think this needs to be fixed ?

```c
    code->constprop = !!(flags & (1 << 4)) | !!(flags & (1 << 5));
```
with `jl_compress_ir` like:
```c
    uint8_t flags =
                  // `code->constprop` requires a special treatment:
                  // `code->constprop & 1 << 5` = none
                  // `code->constprop & 1 << 4` = aggressive
                  | (!!(code->constprop & 2) << 5)
                  | (!!(code->constprop & 1) << 4)
                  | (code->inferred << 3)
                  ...
```

Reply:
```suggestion
    code->constprop = (flags >> 4) & 3;
```
?